### PR TITLE
[SPARK-37956][DOCS] Add Python and Java examples of Parquet encryption in Spark SQL to documentation

### PR DESCRIPTION
**Pull request by andersonm-ibm**
### What changes were proposed in this pull request?
Add Java and Python examples based on the Scala example for the use of Parquet encryption in Spark.

### Why are the changes needed?
To provide information on how to use Parquet column encryption in Spark SQL in additional languages: in Python and Java, based on the Scala example. 

### Does this PR introduce _any_ user-facing change?
Yes, it adds Parquet encryption usage examples in Python and Java to the documentation, which currently provides only a scala example.

### How was this patch tested?
SKIP_API=1 bundle exec jekyll build

Python tab:
image

Java tab:
image

CC @ggershinsky

### CODE REVIEW

File: docs/sql-data-sources-parquet.md [hunk 1]
```diff
@@ -288,6 +290,64 @@ val df2 = spark.read.parquet("/path/to/table.parquet.encrypted")
...
+Dataset<Row> df2 = spark.read().parquet("/path/to/table.parquet.encrypted");
+
+{% endhighlight %}
+
+</div>
+
+<div data-lang="python"  markdown="1">
+{% highlight python %}
+
+sc._jsc.hadoopConfiguration().set("parquet.encryption.kms.client.class" ,
```
Comment (HyukjinKwon) [inline docs/sql-data-sources-parquet.md:325]: Just dropping a comment: we shouldn't use `_jsc`. This isn't an API.

**Comment by andersonm-ibm** [comment]
@HyukjinKwon  Thank you for your review. Your comment was addressed. Could you please have another look?

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Comment by dongjoon-hyun** [comment]
Thank you, @andersonm-ibm , @HyukjinKwon , @srowen .
Can we have this at branch-3.3 too?

cc @MaxGekk

**Comment by srowen** [comment]
We usually don't back port improvements, and this is relatively speaking quite minor among improvements.

**Pull request closed without merging**